c2rust-transpile/src/translator/builtins.rs

```diff
@@ -646,16 +646,9 @@ impl<'c> Translation<'c> {
     ) -> TranslationResult<WithStmts<Box<Expr>>> {
         let args = self.convert_exprs(ctx.used(), args)?;
         args.and_then(|args| {
-            let mut args = args.into_iter();
-            let a = args
-                .next()
-                .ok_or("Missing first argument to convert_overflow_arith")?;
-            let b = args
-                .next()
-                .ok_or("Missing second argument to convert_overflow_arith")?;
-            let c = args
-                .next()
-                .ok_or("Missing third argument to convert_overflow_arith")?;
+            let [a, b, c]: [_; 3] = args
+                .try_into()
+                .expect("`convert_overflow_arith` must have exactly 3 arguments");
```

Comment:
This changes the behavior to require exactly 3 args, whereas previously it required just at least 3 args.  That's probably the behavior we want, though.  @fw-immunant, thoughts?